target/linux/ramips/mt7620/base-files/etc/board.d/01_leds

```diff
@@ -81,6 +81,10 @@ dlink,dwr-960)
 	ucidef_set_led_switch "lan" "lan" "green:lan" "switch0" "0x2e"
 	ucidef_set_led_switch "wan" "wan" "green:wan" "switch0" "0x01"
 	;;
+dlink,dwr-961-a1)
+	ucidef_set_led_switch "lan" "lan" "green:lan" "switch0" "0x3c"
```

Comment:
Just curious, does this handle lan5 too or not?

Reply:
It works for all four lan ports.  `lan5` from internal switch (marked as switch1) is only used as rmgii mac.

Reply:
Oh, right. It has only 4 LAN ports. Stupid question.
Another dumb question, does it use both rgmii1 and rgmii2?

Reply:
Nope. I removed `&rgmii2`.